packaging/linux/post_install.sh

```diff
@@ -18,20 +20,34 @@ optDeprecated="/opt/keybase/mount-readme"
 chown root:root "$krbin"
 chmod 4755 "$krbin"
 
+disableRedirector="false"
+if [ -r "$rootConfigFile" ] ; then
+  if keybase -c "$rootConfigFile" config get -d "$disableConfigKey" &> /dev/null ; then
+    disableRedirector="$(keybase -c "$rootConfigFile" config get -d "$disableConfigKey" 2> /dev/null)"
```

Comment:
Why does this work without the `-b` flag? Won't it give you quotes?

Reply:
No, because the value of the disable key is a json boolean, not a string, and the non-bare quotes only happen for strings.

Reply:
Ah, I keep forgetting that JSON has native booleans.